app/src/providers/property/mapMarkerDataParser.ts

```diff
@@ -0,0 +1,24 @@
+import _ from "lodash";
+import { Property } from "api/codegen";
+
+import { PropertyMapMarker } from "app/map/Map.types";
+
+export function mapMarkerPropertiesDataParser(properties: Array<Property>): Array<PropertyMapMarker> | [] {
+  if (!properties?.length) return [];
+
+  const parsedPropertiesInfo = properties?.map((property) => {
+    const title = property?.content.title;
+
+    return {
+      key: `property_${_.uniqueId(title)}`,
+      title,
+      shortDescription: property?.content.customFields.shortDescription,
```

Comment:
```suggestion
      shortDescription: property.content.customFields.shortDescription,
```

`property` should be truthy now.